frame/support/procedural/src/pallet/parse/mod.rs

```diff
@@ -62,6 +66,25 @@ pub struct Def {
 }
 
 impl Def {
+	fn mark_benchmarking_cached_calls(storage_def: &mut StorageDef, item: &mut syn::Item) {
+		if let syn::Item::Type(typ) = item {
+			for attr in typ.attrs.as_slice() {
+				if attr.style == AttrStyle::Outer {
+					if let Some(seg) = attr.path.segments.last() {
+						if let Ok(_) = syn::parse2::<keyword::benchmarking>(seg.to_token_stream()) {
+							if let Ok(group) = syn::parse2::<Group>(attr.tokens.clone()) {
+								if let Ok(_) = syn::parse2::<keyword::cached>(group.stream()) {
+									storage_def.benchmarking_cached = true;
```

Comment:
What should possibly happen is to have this function return a `syn::Result<bool>` instead of making it mutable upon `storage_def`, then we'll have a cleaner implementation.

Reply:
This has been re-worked to work like other pallet macros. So now it is just `#[pallet::whitelist_storage]` so this issue no longer exists